addons/flashlights/CfgWeapons.hpp

```diff
@@ -22,9 +11,11 @@ class CfgWeapons {
         scope = 2;
         class ItemInfo: InventoryItem_Base_F {
             mass = 1;
-            class FlashLight {
-                ACE_Flashlight_Colour = "red";
-                ACE_Flashlight_Size = 1.75;
+            class ACE_Map_Flashlight {
```

Comment:
I don't see why renaming this is necessary. The entries are already prefixed anyways.